aries-site/src/components/feedback/FeedbackButton.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+import { Button } from 'grommet';
+
+export const FeedbackButton = styled(Button)`
```

Comment:
Something like this feels more Grommet-y:

```
const PositionedBox = styled(Box)`
  position: fixed;
  bottom: 0px;
  right: 0px;
  z-index: 9999;
  }
`;

export const FeedbackButton = ({ elevation, margin, ...buttonProps }) => (
  <PositionedBox elevation={elevation} margin={margin}>
    <Button {...buttonProps} />
  </PositionedBox>
);

```

Reply:
☝️ This is one of the rare cases where I'd actually use margin, because it is still being self-centered and not concerned about others. It is saying "don't crowd me."

Reply:
Would we put this in grommet though? feels like it was a unique case for our site. But I can adjust the code to be cleaner.